addons/mail/static/src/components/call_main_view/call_main_view.xml

```diff
@@ -4,7 +4,6 @@
 <t t-name="mail.CallMainView" owl="1">
     <t t-if="callMainView">
         <div class="o_CallMainView d-flex flex-grow-1 flex-column align-items-center justify-content-center position-relative bg-black-50" t-ref="root" t-on-mouseleave="callMainView.onMouseleave">
-            <t t-set="dummy" t-value="messaging and messaging.device and messaging.device.globalWindowInnerHeight and messaging.device.globalWindowInnerWidth"/>
```

Comment:
What allows to remove this?

Reply:
it was the previous way of observing the window resize, but now that we observe the element resize, we don't need it anymore